tests/utest/zap/test_basic_functionality.py

```diff
@@ -300,7 +300,17 @@ def assertNotNoneOrEmpty(self, str_):
         return str_ is not None and str_ != ''
 
     def test_cli(self):
```

Comment:
We need more unit tests for cli:
- `--accepted-risk-level` given, `--required-confidence-level` not given
- `--required-confidence-level` given, `--accepted-risk-level` not given
- neither of the optional arguments given

This makes sure that someone accidentally does not change the optional argument to a mandatory one.